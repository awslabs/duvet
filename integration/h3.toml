name = "h3"
repo = "https://github.com/hyperium/h3"
version = "c6b92cbca902a62850b72269384fcbc32d30cb96"
cmd = [
    "bash ci/compliance/extract.sh",
    "bash ci/compliance/report.sh",
]
html_report = "ci/compliance/report.html"
