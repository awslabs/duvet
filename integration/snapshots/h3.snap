version https://git-lfs.github.com/spec/v1
oid sha256:ad8e8048f7907ce0b5fe5447ef67a1e37cae42700d48406d11d4dd74cf96a8c5
size 690627
